src/MockBot.Api/Controllers/ChatController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MockBot.Api.Interfaces;
+using MockBot.Api.Models;
+
+namespace MockBot.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ChatController : ControllerBase
+    {
+        private readonly IChatService _chatService;
+
+        public ChatController(IChatService chatService)
+        {
+            _chatService = chatService;
+        }
+
+        [HttpGet]
+        public IEnumerable<Chat> FindAll()
+        {
+            return _chatService.FindAll();
+        }
+
+        [HttpGet("{id:guid}")]
```

Comment:
I'm not sure you need to add the `("{id:guid}")` bit as the model binding will already be able to differentiate between the two `Get` methods based on the parameters

Reply:
This is something Rider recommended, I usually just Alt-Enter until the IDE is happy. Can remove if you'd like.

Reply:
I'd remove it and check it still works, if it does leave it out.

By defining it in the attribute and method signature, we have two places to change it which does not make a lot of sense to me.

.net will take care of this for you based on the method signature, so it could be best to let the framework do its job ;)

Reply:
I have removed these